modules/globebrowsing/globebrowsingmodule.cpp

```diff
@@ -170,6 +181,9 @@ namespace {
         // [[codegen::verbatim(TileCacheSizeInfo.description)]]
         std::optional<int> tileCacheSize;
 
+        // [[codegen::verbatim(DefaultGeoPointTextureInfo.description)]]
+        std::optional<std::string> defaultGeoPointTexture;
```

Comment:
Hm, I agree, but, will this allow setting an empty path?

Reply:
An empty path is still a valid `std::filesystem::path` since it doesn't check for whether the path exists or not (something that would be nice to add into codegen as an option though (https://github.com/OpenSpace/OpenSpace/issues/2602)

Reply:
Although, using `std::filesystem::path` here means that we can't use path tokens to set the property value (leads to a "file idd not exists" specification error). Which we would want, at least for the module since that paht is set in the .cfg file, but IMO we would want this to work for all filepaths we enter as properties. 

So I'll keep it as a string for now? (And make the DefaultProperties texture back to string as well?)